### ISSUE TITLE
[accessibility] files: file list actions not focusable

### ISSUE THREAD

**Issue by susnux** [nextcloud/server#62246]
After selecting one or all checkboxes in front of files/folders, the "Add to Favorites" and "Move and Copy" buttons appear, obscuring the column headers. These buttons are not accessible via keyboard shortcuts.

**Comment by susnux** [comment]
https://github.com/nextcloud/server/issues/58386
